sites/default/files/css_editor/bootstrap.css

```
@@ -983,74 +1061,88 @@ p.support-text {
     min-height: 70px;
 }
 
-.field--name-field-case-study-subtitle{
-  	background-color: #03396C;
-    font-size: 1.5em;
-    text-align: left;
-  	color: #FFFFFF;
-  	border-radius: 0px 0px 0px 0px;
-  	padding-left: 1em;
-  	padding-top: 1em;
+.page-node-type-case-study .block-field-blocknodecase-studytitle>span{
+  font-weight: bolder;
+  font-size: 36px;
+  text-align: left;
+  background-color: #03396C;
+  color: #FFFFFF;
+  border-radius: 0px 0px 0px 0px;
+  padding-left: 4em;
+  padding-right: 4em;
+  padding-top: 20px;
+  padding-bottom: 0.2em;
+  margin-left: -4em;
+  margin-right: -4em;
+  word-wrap: break-word;
 }
-.case-study .page-header{
-    font-size:300%;
-    font-weight:bolder;
-    	background-color: #03396C;
-    text-align: left;
-  	color: #FFFFFF;
-  	border-radius: 0px 0px 0px 0px;
-  	padding-left: 1em;
-  	padding-top: 1em;
+
+.field--name-field-case-study-subtitle{
+  background-color: #03396C;
+  font-size: 1.5em;
+  text-align: left;
+  color: #FFFFFF;
+  border-radius: 0px 0px 0px 0px;

```

```suggestion
  border-radius: 0;
```
Could just be this, but not to bothered about stuff like that. Just FYI.